bk-api/bk_api.py

```diff
@@ -42,28 +42,11 @@
 import requests
 import re
 
-from sshkeygen import SSHKeyGen, run_command, run_command_communicate
-import traceback
-from werkzeug.utils import secure_filename
+from sshkeygen import SSHKeyGen, run_command_communicate
 
 import linecache
 
-
-formatter = logging.Formatter(
-    "%(asctime)s  [%(name)s:%(lineno)d] (%(levelname)s): %(message)s"
-)
-handler = logging.StreamHandler(stream=sys.stdout)
-handler.setFormatter(formatter)
 logger = logging.getLogger(__name__)
-logger.setLevel(logging.DEBUG)
-logger.addHandler(handler)
-
-
-
-
-for k, v in sorted(os.environ.items()):
```

Comment:
I removed this as you may accidentally print some sensitive env var like a token or password to the logs.

Reply:
sounds good!